.circleci/config.yml

```diff
@@ -12,16 +12,26 @@ integrationDefaults: &integrationDefaults
     - KUBECONFIG: /go/out/minikube.conf
     - TEST_ENV: minikube-none
 
-# remote docker environment. Appears to be faster
-integrationRemote: &integrationRemote
+# remote docker environment for one namespace. Appears to be faster
+integrationRemoteOneNamespace: &integrationRemoteOneNamespace
```

Comment:
The real solution is to make sure the tests are compatible with multiple namespaces. 
How about we keep the 'one namespace' option only for demo ? I don't think we need more tests for one namespace
since it'll not be recommended.

Reply:
There are some gaps that will break `isolated-namespaces` case if we want to reuse the e2e test framework, eg. `old-integration`. 
Fortunately, we get all current test cases passed.

Reply:
I would still like to make it clear this is a 'demo' profile only - to avoid confusion. I think it is very important for security and operations to move to separate namespace and stop supporting 'one namespace'.